src/modules/iotjs_module_buffer.c

```diff
@@ -252,6 +252,17 @@ JHANDLER_FUNCTION(Compare) {
 }
 
 
+#define CONVERT_DOUBLE_TO_SIZE(n, d) \
+  size_t n;                          \
+  do {                               \
+    if (d < 0) {                     \
```

Comment:
@LaszloLango Thank you for nice patch. I have probably last question. When d is +inf or -inf, and NaN, how will it work? If you know, let me know. Thank you again.